EdFi.Ods.Utilities.Migration.Tests/MsSql/MigrationTests/v33_to_v34/V33ToV34SqlServerMigrationTest.cs

```diff
@@ -47,5 +53,23 @@ protected OdsUpgradeResult PerformTestMigration(string sourceDataScriptName = nu
         {
             return GetTableContents<T>(ToVersion);
         }
+
+        [Test]
+        public void ValidateJournalEntries()
+        {
+            var databaseReferencesJournalEntries = FetchDatabaseReferencesJournalEntries().ToList();
+
+            PerformTestMigration();
+
+            var deployJournalFullList = GetTableContents<DeployJournal>("[dbo].[DeployJournal]").Select(
+                x => x.ScriptName);
+
+            var deployJournal34List = deployJournalFullList.Where(y => databaseReferencesJournalEntries
+                    .Any(z => z.Contains(y,StringComparison.OrdinalIgnoreCase)))
+                    .ToList();
+
+            databaseReferencesJournalEntries.ToHashSet().SetEquals(deployJournal34List.ToHashSet()).ShouldBeTrue(
+                $"The JournalEntries scripts did not match the scripts available to the Migration Utility for  version {ToVersion.DisplayName}.");
```

Comment:
It might be useful to list the missing entries here similar to the check in `EdFiOdsVersionJournalTests`.